zebra/rt_netlink.c

```diff
@@ -574,7 +576,7 @@ static int netlink_route_change_read_unicast(struct nlmsghdr *h, ns_id_t ns_id,
 	if (rtm->rtm_protocol == RTPROT_KERNEL)
 		return 0;
 
```

Comment:
I'd put line 536 here instead the point being delaying work until we need to do it since we are testing a bunch of other stuff and returning before this point.

Reply:
Made the requested changed. PTAL